agent/app/agent_capability_windows.go

```diff
@@ -16,10 +16,22 @@
 package app
 
 import (
+	"path/filepath"
+
+	"github.com/aws/amazon-ecs-agent/agent/config"
 	"github.com/aws/amazon-ecs-agent/agent/ecs_client/model/ecs"
 	"github.com/aws/amazon-ecs-agent/agent/taskresource/volume"
 )
 
+var (
+	capabilityDepsRootDir          = filepath.Join(config.AmazonProgramFiles, "ECS\\managed-agents")
```

Comment:
nit: I'd rather change `config.AmazonProgramFiles` to `config.AmazonECSProgramFiles`. And then in `config_windows.go` do: `AmazonECSProgramFiles = filepath.Join(envProgramFiles, "Amazon", "ECS")`. The agent is ECS specific so this naming feels more appropriate.

Reply:
done